packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -150,6 +146,23 @@ def _setup_logging():
     logging.getLogger("requests").setLevel(logging.WARNING)
 
 
+@contextlib.contextmanager
```

Comment:
I don't think that's a good fit for a util function. We'd have to pass the `monkeypatch` object from the test just so this function could use it, or use it within the test and leak this function's implementation details. `monkeypatch` is intended for mocking things used by code under test, but here we're not testing boto, just trying to get it to use the right credentials. Coupling this function to pytest would make it harder to extract. I think the cost of reinventing this (small) wheel outweighs the benefits.